pkg/cmd/server/origin/admission/plugin_initializer.go

```diff
@@ -144,6 +145,14 @@ func NewPluginInitializer(
 		cloudConfig,
 		restMapper,
 		quotaRegistry)
+	// upstream broke this, so we can't use their mechanism.  We need to get an actual client cert and practically speaking privileged loopback will always have one
+	kubePluginInitializer.SetClientCert(privilegedLoopbackConfig.TLSClientConfig.CertData, privilegedLoopbackConfig.TLSClientConfig.KeyData)
```

Comment:
Where can a webhook find the CA that signs this cert?

Reply:
> Where can a webhook find the CA that signs this cert?

This cert is signed by the client CA for the apiservers.  Since the our webhook apiservers are run as aggregated API servers, they automatically trust that CA (though a user could wire it differently).  That means that normal cluster authentication and authorization (rbac) rules apply.